### DISCUSSION TITLE
GDPR and banning info

### DISCUSSION THREAD

**Discussion by WikiRik** [benawad/dogehouse#1241] [General]
I just saw the new DogeLog and started Googling. It seems that GDPR allows for maintaining minimal data if there is a legitimate interest for it. So keeping only the account ID and the bans in place is allowed.

But I'm not a lawyer and just did a little bit of searching. So I'm eager to hear from others what they have found about this topic, and also maybe about related privacy laws in other countries/regions.

Main source I used: https://www.reddit.com/r/gdpr/comments/7wegg0/gdpr_and_banning_users/du052s8/

**Comment by dk-raw** [comment]
We are still working on the legal stuff.
